### PR TITLE
Added in support for npm publish

### PR DESCRIPTION
**Pull request by boomanaiden154**
This PR adds in support for pushing releases to NPM.  No changes are needed for `mathquill4quill.js`, so all that needed to be done was to add in a name in `package.json`, and update `.release-it.json` to push to npm. I pushed to the npm repository(with the wrong version, need to fix that), and @c-w, I can invite add you as a maintainer to the package at npm. I verified this working with browserify, and I am sure it will work with webpack too.

Resolves https://github.com/c-w/mathquill4quill/issues/16

### CODE REVIEW

**Comment by boomanaiden154** [comment]
I updated the package on npm to have the current version(0.1.3), and unpublished the incorrect version(0.1.5).

File: .release-it.yaml [hunk 1]
```diff
@@ -1,17 +0,0 @@
-git:
```
Comment (c-w) [inline .release-it.yaml]: Any specific reason for moving this from yaml to json? Could we keep it in yaml for less syntax overhead?

Comment (boomanaiden154) [reply .release-it.yaml]: I moved it to JSON because it better matched the documentation of release-it, but I can change it back if you prefer.

Comment (c-w) [reply .release-it.yaml]: In that case, JSON is fine by me. Thanks for the update.

**Comment by c-w** [comment]
@boomanaiden154 My NPM username is [c-w](https://www.npmjs.com/~c-w), could you please add me as a maintainer on the package? Thanks!

**Comment by boomanaiden154** [comment]
@c-w I just added you as a maintainer on the package.

**Pull request merged**